.github/workflows/branch_release.yml

```diff
@@ -0,0 +1,24 @@
+# Workflow to automate creation and updating of release branches
+name: Release Branching
+
+# Trigger on tags created by TeamCity
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  branch_release:
+    if: github.event.created && github.event.sender.login == 'labkey-teamcity'
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Create branches and PRs
+      uses: labkey-tchad/gitHubActions/branch-release@master
```

Comment:
Are these actions going to move somewhere else, not associated with a particular user?